internal/input/input_glfw.go

```diff
@@ -22,13 +22,16 @@ package input
 import (
 	"sync"
 	"unicode"
+	"time"
 
 	glfw "github.com/go-gl/glfw/v3.2/glfw"
 )
 
 type Input struct {
 	keyPressed         map[glfw.Key]bool
 	mouseButtonPressed map[glfw.MouseButton]bool
+	scrollY			   float64
+	scrollTime		   time.Time
```

Comment:
Please run `gofmt -s -w .` or the CI test doesn't pass.

Reply:
Thx, that solved that problem.